src/Administration/Resources/app/administration/src/module/sw-order/page/sw-order-detail/index.js

```diff
@@ -549,8 +508,35 @@ export default {
             return this.orderRepository.save(this.order, this.versionContext);
         },
 
+        handleCartErrors(response) {
+            if (!response?.data?.errors) {
+                return;
+            }
+
+            Object.values(response?.data?.errors).forEach(({ level, message }) => {
```

Comment:
Not needed, because of if before
```suggestion
            Object.values(response.data.errors).forEach(({ level, message }) => {
```